docs/modules/ROOT/pages/usage.adoc

```diff
@@ -47,31 +38,10 @@ spec:
             rootLogLevel: INFO
         replicas: 3
 ----
-
-If you want to set a password for the initial admin user you can do this by applying the following object:
-
-[source,yaml]
-----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: nifi-admin-credentials-simple
-stringData:
-  username: admin
-  password: supersecretpassword
-----
-
-You can create the ZNode config map referenced in `zookeeperConfigMapName` via (assuming there is a ZooKeeper cluster called `simple-zk`:
-[source,yaml]
-----
-apiVersion: zookeeper.stackable.tech/v1alpha1
-kind: ZookeeperZnode
-metadata:
-  name: simple-nifi-znode
-spec:
-  clusterRef:
-    name: simple-zk
-----
```

Comment:
I did this because:
* the getting started guide already has _complete_ examples of these resources. For example no ZK cluster is created here.
* I explained in the NifiCluster how admin credentials and znode are handled and this would have been redundant.
* imo the usage page should focus on how to **use** the NifiCluster resource without having to start from scratch every time.